access/src/main/external/static/css/admin/jquery-ui.css

```diff
@@ -955,6 +955,7 @@ body .ui-tooltip {
 
 .ui-icon-close { background-position: -80px -128px; }
 .ui-icon-closethick { background-position: -96px -128px; }
+.ui-icon-check { background-position: -64px -144px; }
```

Comment:
Is this a patch to this version of jquery-ui, or has the CSS been updated to be in sync with the jquery-ui JavaScript?